packages/babel-plugin-polyfill-corejs3/src/shipped-proposals.ts

```diff
@@ -4,13 +4,6 @@ export default new Set<string>([
   "esnext.array.from-async",
   "esnext.array.group",
   "esnext.array.group-to-map",
-  "esnext.array.to-reversed",
-  "esnext.array.to-sorted",
-  "esnext.array.to-spliced",
-  "esnext.array.with",
   "esnext.string.is-well-formed",
   "esnext.string.to-well-formed",
```

Comment:
Note that it can be removed by automatic build because of a bug in WebKit - but it's still a shipped proposal.